_drafts/rse-at-egu.md

```diff
@@ -0,0 +1,73 @@
+The need for outreach also applies to holders of an office within the union, to reach better acknowledgements of the needs of RSEs in universities, scientific unions, and at scientific conferences.
+
+Some good ideas came up and we plan to reach out to EGU and ESSI leadership and share them.
+For example, EGU could increase recognition of RSEs and their work by awarding a medal, by offering a special poster track (not falling under submission limitations) for contributions to scientific software, or by tagging abstracts as "RSE" similarly to the "ECS" badges.
+These are "top-down" ideas that we would like towards within the organisation.
+The potential and usefulness of such domain-specific (taking all EGU members as a group for a moment) actions, who are lateral to the national chapters, was commonly agreed on.
+But there were also "bottom-up" actions who promise to support the RSE organisation's causes, such as scientists suggesting sessions related to RSE roles and activities within their divisions, offering short courses specifically around RSEng capabilities (and also labeling the numerous existing courses as such), or organising software and [data carpentry courses](https://carpentries.org/) in the week before or after the conference.
+If you think about convening a session on scientific software, please [get in touch](mailto:daniel.nuest@uni-muenster.de)!
+Who would not want to join _"Software development for professors - what does my student do with his computer"_?
+Jokes apart, it was great to follow the lively discussion, which was enriched with a common understanding of the values and importance of diversity and openness.
+
+An important aspect of structured RSE activities are surveys, because the people self-identifying as RSE are diverse and wide-spread and the roles that RSEs play in scientific research are manifold.
+We want to contribute to the process of understanding the needs and diversity of people involved in research software engineering with the following survey on EGU attendees:
+
+**[ADD SURVEY LINK](ADD SURVEY LINK)**
+
+The townhall meeting was a good start to spreading the word about the goals of RSE organisations and the activities to put research software engineering on the map of all stakeholders in science, such as researchers, publishers, funding agencies, and scientific unions.
+What can we do better next year?
+Hopefully we can do what more "established" townhalls can offer: snacks and drinks!
+We should also announce and prominently place a sticker table.
+Apart from that, the townhall did an excellent job its first time, just like other long-running townhall meetings at EGU GA: it provided a place for like-minded people to connect and for newcomers to dip their toes into a new topic and be introduced by dedicated members of an international friendly community of people who "do science with code".
```

Comment:
"Apart from that, this first townhall did an excellent job, just like ... : It provided"  (capital "It" after colon)

"be introduced to"? Dependening on whether the newcomers were introduced to the community by dedicated members, or if they were introduced to dedicated members...